lib/common/messages.c

```diff
@@ -115,11 +115,6 @@ create_reply_adv(const xmlNode *original_request, xmlNode *xml_response_data,
         crm_err("Cannot create new_message, no message type in original message");
         CRM_ASSERT(type != NULL);
         return NULL;
-#if 0
-    } else if (strcasecmp(XML_ATTR_REQUEST, type) != 0) {
-        crm_err("Cannot create new_message, original message was not a request");
-        return NULL;
-#endif
```

Comment:
The idea is that we have request messages and reply messages, and replies should only be generated for requests. Unfortunately in practice we have some odd usages here and there, so we probably can't enforce it. (I know we have issues the other way -- some replies are created as requests.) Maybe change this to a trace message with a comment and don't return.

Reply:
I can, but it'll still be in a #if 0 block which is why I was looking at removing it in the first place.

Reply:
I mean drop the #if, and have it just log a trace message (not return), with a comment that requiring a match might be a good idea if we can ensure everything complies